### DISCUSSION TITLE
Welcome to trs80mc10 Discussions!

### DISCUSSION THREAD

**Discussion by jggames** [jggames/trs80mc10#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jggames** [comment]
This is a test of the discussion system. This is only a test...

**Comment by gregdionne** [comment]
This is a test of the Discussion System. The retroprogrammers of your area
in voluntary cooperation with the local MC-10 authorities have developed
this system to keep you informed in the event of something even remotely
mildly interesting. If this had been an actual discussion, the Forum
Message you just read would have been followed by official shenanigans,
programs or blather. This forum serves the trs80mc10 area. This concludes
this test of the Discussion System."
